tests/network/bindingplugin.go

```diff
@@ -100,6 +104,65 @@ var _ = SIGDescribe("[Serial]network binding plugin", Serial, decorators.NetCust
 			Expect(vmi.Status.Interfaces[0].MAC).To(Equal(macAddress))
 		})
 	})
+
+	Context("macvtap", func() {
+		const (
+			macvtapNetworkConfNAD = `{"apiVersion":"k8s.cni.cncf.io/v1","kind":"NetworkAttachmentDefinition","metadata":{"name":"%s","namespace":"%s", "annotations": {"k8s.v1.cni.cncf.io/resourceName": "macvtap.network.kubevirt.io/%s"}},"spec":{"config":"{ \"cniVersion\": \"0.3.1\", \"name\": \"%s\", \"type\": \"macvtap\"}"}}`
+			macvtapBindingName    = "macvtap"
+			macvtapLowerDevice    = "eth0"
+			macvtapNetworkName    = "net1"
+		)
+
+		createMacvtapNetworkAttachmentDefinition := func(namespace, networkName, macvtapLowerDevice string) error {
+			macvtapNad := fmt.Sprintf(macvtapNetworkConfNAD, networkName, namespace, macvtapLowerDevice, networkName)
+			return createNetworkAttachmentDefinition(kubevirt.Client(), networkName, namespace, macvtapNad)
+		}
```

Comment:
Can you please reuse or do something similar to `createBasicNetworkAttachmentDefinition`, this one is too noisy. You know I do not like these closure functions :).

Reply:
removed it.